tokio/src/fs/file.rs

```diff
@@ -233,13 +233,14 @@ impl File {
     /// let std_file = std::fs::File::open("foo.txt").unwrap();
     /// let file = tokio::fs::File::from_std(std_file);
     /// ```
-    pub fn from_std(std: StdFile) -> File {
+    pub fn from_std(mut std: StdFile) -> File {
+        let pos = std.stream_position().unwrap();
```

Comment:
I think we probably want to just fall back to existing behavior if `stream_position` fails.
```suggestion
        let pos = std.stream_position().unwrap_or(0);
```